corehq/apps/app_manager/management/commands/migrate_inline_search_instance.py

```diff
@@ -0,0 +1,59 @@
+import re
+
+from corehq.apps.app_manager.management.commands.helpers import (
+    AppMigrationCommandBase,
+)
+from corehq.toggles import USH_INLINE_SEARCH
+
+old_instance = "search_results"
```

Comment:
this is `search_results` because I already ran this once on staging to migrate from `results` to `search_results`